roles/tendrl-node-agent/tasks/main.yml

```diff
@@ -19,6 +19,13 @@
     regexp='^etcd_connection:*'
     line="etcd_connection{{ ':' }} {{ groups['usm_server'][0] }}"
 
+- name: Add provisioner/ceph tag in node-agent.conf.yaml (on tendrl-server)
+  lineinfile:
+    dest=/etc/tendrl/node-agent/node-agent.conf.yaml
+    insertafter='^tags:$'
+    line="  - provisioner/ceph"
+  when: "'usm_server' in group_names"
```

Comment:
Could you add a reference to Tendrl documentation or commit for this feature/configuration?